.travis.yml

```diff
@@ -19,6 +19,7 @@ cache:
 
 deploy:
 - provider: s3
+  edge: true
```

Comment:
silly me, skip_cleanup was deprecated in dpl v2, but we were still using v1

We can opt in to v2 and try. If we still have problems, we can revert to v1 for now.